signaturit-sdk.gemspec

```diff
@@ -15,7 +15,11 @@ Gem::Specification.new do |s|
 
     s.license     = 'MIT'
 
-    s.add_dependency 'rest-client', '~> 1.7'
+    s.add_dependency 'rest-client', '> 1.7'
```

Comment:
I think that:

~> 1.7 is for 1.7x, 1.8x, etc until 2.x
>= 1.7 is for 1.7x, 1.8x, etc including 2.x, 3.x etc

If that's ok, I think that ~> is a better option. Do you agree?